uast/diff/testdata/create_testdata.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
```

Comment:
The script is pretty simple. Can be written in Go as well (it's the only Python file here).

Reply:
I only threw it in so that one may try to regenerate/generate new test cases. Also, see the answers below.

Reply:
I still think it makes sense to rewrite it in Go. All the SDK is in Go. I see no reason to require Python installation for such a simple script.